plugins/wikipedia.js

```diff
@@ -34,10 +34,12 @@ function wikipedia(title) {
                 const json = JSON.parse(body);
 
                 if (json.query) {
-                    const extract = values(json.query.pages)[0].extract;
+                    const extract = values(json.query.pages)[0].extract,
+                          pageId = values(json.query.pages)[0].pageid,
+                          pageUrl = 'https://en.wikipedia.org/?curid=' + pageId;
 
                     if (extract) {
-                        resolve(extract.split('\n')[0]);
+                        resolve(extract.split('\n')[0] + ' ' + pageUrl);
```

Comment:
Actually I think it's better to keep it that way since some of the API replies are usually one lines which will look a bit odd to always put the link under the text in all cases. And a bit of an overkill to check wether its a long reply or a short one IMO. Here is an example:

<img width="1045" alt="screen shot 2017-03-25 at 11 09 22 am" src="https://cloud.githubusercontent.com/assets/823713/24321052/b63a97e6-114b-11e7-98e3-67819c17bb3e.png">